selfify/sidebar/choose_file.js

```diff
@@ -0,0 +1,63 @@
+/*
+Listens for a file being selected, creates a ObjectURL for the chosen file, injects a
+content script into the active tab then passes the image URL through a message to the
+active tab ID.
+*/
+
+// Listen for a file being selected through the file picker
+let inputElement = document.getElementById("input");
```

Comment:
Sorry I should have been clearer. I didn't mean, consider making *this one `var`* a `let` or `const`.

I meant, in general, in this source file, and the others, *consider* whether using `const` or `let` would be better. I think it's OK to use `var` everywhere, so if that's your preference I think it's fine. But `const` or `let` are probably better practice in many situations (personally I would prefer to use `const` in this particular case, where I'm storing a reference to a node). 

However, I don't think changing *just this one* `var` to `let` is an improvement: it's better to have a style and stick to it consistently.